api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -60,7 +60,10 @@
     private static final LocalDate TURNS_FOUR_IN_FIRST_WEEK_OF_NEXT_PAYMENT_CYCLE = START_OF_NEXT_CYCLE.minusYears(4).plusDays(4);
     private static final LocalDate SIX_MONTH_OLD = LocalDate.now().minusMonths(6);
     private static final LocalDate THREE_YEAR_OLD = LocalDate.now().minusYears(3);
+    private static final List<LocalDate> SINGLE_THREE_YEAR_OLD = singletonList(THREE_YEAR_OLD);
     private static final int CARD_BALANCE_IN_PENCE_BEFORE_DEPOSIT = 88;
+    private static final LocalDate EXPECTED_DELIVERY_DATE = LocalDate.now().plusMonths(4);
```

Comment:
Is it worth renaming this to something like `DUE_DATE_IN_4_MONTHS`?

Reply:
Yep, sure thing.